src/main/java/com/marklogic/rest/util/RestTemplateUtil.java

```diff
@@ -50,6 +52,21 @@ public boolean isTrusted(X509Certificate[] chain, String authType) throws Certif
 					}
 				}).build();
 				httpClientBuilder.setSslcontext(sslContext);
+				httpClientBuilder.setHostnameVerifier(new X509HostnameVerifier() {
```

Comment:
Could you use DatabaseClientFactory.SSLHostnameVerifier.ANY here instead? 

I'll publish alpha4 with this regardless.